design/FY2023/NFP_MultispeedFans.md

```diff
@@ -0,0 +1,817 @@
+       \type choice
+       \key Coil:Heating:Fuel
+       \key Coil:Heating:Electric
+       \key Coil:Heating:Water
+       \key Coil:Heating:Steam
+       \note works with gas, electric, hot water and steam heating coil.
+  	A18, \field Supplemental Heating Coil Name
+       \type object-list
+       \object-list HeatingCoilName
+       \note Needs to match in the supplemental heating coil object.
+  	N11, \field Maximum Supply Air Temperature from Supplemental Heater
+       \required-field
+       \type real
+       \units C
+       \autosizable
+       \default autosize
+       \note Supply air temperature from the supplemental heater will not exceed this value.
+  	N12, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
+       \type real
+       \maximum 21.0
+       \default 21.0
+       \units C
+       \note Supplemental heater will not operate when outdoor temperature exceeds this value.
+  	A19, \field Controlling Zone or Thermostat Location
+       \type object-list
+       \object-list ZoneNames
+       \note Used only for AirloopHVAC equipment on a main branch and defines zone name where thermostat is located.
+       \note Not required for zone equipment. Leave blank if terminal unit is used in AirLoopHVAC:OutdoorAirSystem:EquipmentList.
+       \note Required when terminal unit is used on main AirloopHVAC branch and coils are not set point controlled.
+       \note When terminal unit is used in air loop and is load controlled, this zone's thermostat will control operation.
+
+<span style="color:red">
+
+  	A20, \field Design Specification Multispeed Fan Object Type
+       \type choice
+       \key FanPerformance:Multispeed
+       \note Enter the type of performance specification object used to describe the multispeed fan.
+  	A21; \field Design Specification Multispeed Fan Object Name
+       \type object-list
+       \object-list FanPerformaceNames
+       \note Enter the name of the performance specification object used to describe the multispeed fan.
+</span>
+
+###FanPerformance:Multispeed###
+
+This section provides inputs of the new object.
+
+<span style="color:red">
+
+An alternative way is to use the existing object of UnitarySystemPerformance:Multispeed. Since this object is not used only for UnitarySystem, the name of the object may be changed.
+
+It should be pointed out that since this object mainly specify airflow ratios at different speeds, it is OK to use the object with minor name change.
+  
+</span>
+
+
+	FanPerformance:Multispeed,
+       \memo The FanPerformance object is used to specify the air flow ratio at each
+       \memo operating speed. This object is primarily used for ZoneHVAC:WaterToAirHeatPump and 
+       \memo ZoneHVAC:TerminalUnit:VariableRefrigerantFlow objects to allow operation at 
+       \memo different fan flow rates.
+       \extensible:2 - repeat last two fields, remembering to remove ; from "inner" fields.
```

Comment:
@rraustad If this object is truly extensible, then the fields for Number of Speeds for Heating/Cooling should not have `\maximum` declared in the IDD.